src/commands/install-deps.yml

```diff
@@ -0,0 +1,3 @@
+description: "Install gems with Bundler."
+steps:
```

Comment:
These partial files all moved into a `src/` directory and were indented to not be nested under the `description` key. I swear there aren't additional changes 🖐 .